conf/config.yaml

```diff
@@ -49,6 +49,7 @@ data:
               username: root
               password: "123456"
               database: employees_0000
+              parameters: maxAllowedPacket=268435456
```

Comment:
Maybe the format is better:
```
parameters: 
    maxAllowedPacket: 268435456
```

we can add other parameter in the future conveniently.